WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java

```diff
@@ -581,24 +582,26 @@ private void showComment() {
             layoutBottom.setVisibility(View.GONE);
 
             if (mNote != null) {
-                // If a remote comment was requested, check if we have the comment for display.
-                // Otherwise request the comment via the REST API
                 SiteModel site = mSiteStore.getSiteBySiteId(mNote.getSiteId());
-                if (site != null) {
-                    CommentModel comment = mCommentStore.getCommentBySiteAndRemoteId(site, mNote.getParentCommentId());
-                    if (comment != null) {
-                        setComment(comment, site);
-                        return;
-                    }
+                if (site == null) {
+                    ToastUtils.showToast(getActivity(), R.string.error_load_comment);
+                    return;
+                }
 
-                    mCommentIdToFetch = mNote.getParentCommentId() > 0 ? mNote.getParentCommentId() : mNote.getCommentId();
-                    RemoteCommentPayload payload = new RemoteCommentPayload(site, mCommentIdToFetch);
+                // Check if the comment is already in our store
+                CommentModel comment = mCommentStore.getCommentBySiteAndRemoteId(site, mNote.getCommentId());
```

Comment:
I might be missing something here - why did this change from using `mNote.getParentCommentId()` to `mNote.getCommentId()`?